script/reverse_urls.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+from argparse import ArgumentParser
+from urllib.parse import urlsplit
+import re
+
+"""
+>>> reverse_url("http://some.url/to?reverse=true")
+gov.bnl.face.www.facedailyreports.facts_i.archive.03132003.archivepage.htm.
```

Comment:
In python, comments to functions go underneath the declaration line, so
```python
def reverese_url(url):
    """
    >>> ...
    """
    <code>
```
Now that you can run it, it fails:
```
$ python -m doctest script/reverse_urls.py
**********************************************************************
File "script/reverse_urls.py", line 9, in reverse_urls.reverse_url
Failed example:
    reverse_url("http://some.url/to?reverse=true")
Expected:
    gov.bnl.face.www.facedailyreports.facts_i.archive.03132003.archivepage.htm.
Got:
    'url.some.to.reverse.true'
**********************************************************************
1 items had failures:
   1 of   1 in reverse_urls.reverse_url
***Test Failed*** 1 failures.
```
I think you forgot to update the argument.

Reply:
Fixed. Sorry I was editing from Github :)